openedx/core/djangoapps/notifications/views.py

```diff
@@ -283,12 +281,21 @@ def get(self, request):
         for item in count_by_app_name:
             app_name = item['app_name']
             count = item['count']
-
             count_total += count
             count_by_app_name_dict[app_name] = count
-        # Return the unseen notifications count for the user and the unseen notifications count for each app name.
+
+        learner_course_enrollments = CourseEnrollment.objects.filter(
+            user=request.user,
+            is_active=True
+        ).values_list('course__id', flat=True)
+
+        show_notifications_tray_enabled = any(
```

Comment:
'any' will create a list first and then return if at least one element is True. If we iterate through the list via a loop we can break when the first True is returned.